javascript-algorithms-and-data-structures/learn-debugging-by-building-a-TODO-app/index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Learn debugging by building a TODO app</title>
+
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+
+  <body>
+    <h1 class="title">TODO App</h1>
+
+    <main class="app">
+      <button id="open-modal-btn" class="btn open-modal-btn">
+        Add New Task
+      </button>
+
+      <!-- Modal -->
+      <form
+        class="modal"
+        id="modal"
+        tabindex="-1"
+        aria-labelledby="TODOModalLabel"
+        aria-hidden="true"
```

Comment:
We actually don't need to apply the `aria-hidden` property when the `display` is set to `none`.

<https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden>

Also, by applying this and not removing it in the onClick handler, the modal would never be visible to screen readers.

Reply:
ok, cool. That makes sense. 
I will remove it 👍